docs/docs/technical-reference/openrefine-api.md

```diff
@@ -159,9 +159,14 @@ On success returns JSON response
 
 > **Command:** _POST /command/core/export-rows_
 
-      'project' : project id
-      'engine' : JSON string... (e.g. '{"facets":[],"mode":"row-based"}')
+In the parameter
+
+      'project' : project id      
       'format' : format... (e.g 'tsv', 'csv')
+
+In the form data
+
+      'engine' : JSON string... (e.g. '{"facets":[],"mode":"row-based"}')
```

Comment:
As far as I know, parameters can be passed either in the query string or in form data and the API doesn't care. By convention, the Engine config data is usually sent as form data because of its size, but that's just a convention.

Did you run into a case where the parameters could only be passed via on mechanism or the other?

Reply:
For me export-rows API call doesn't work if I pass one of these parameters (format, project) via form data. It works only if I pass these parameters as request parameters. I don't have any issues with engine parameter it works for me either way.

Reply:
There are indeed cases where parameters have to be in the URL or in the POST body.